README.md

```diff
@@ -153,6 +153,16 @@ type ModelSomeUserInfoModel struct {
 }
 ```
 
+If you want to add tags also, i.e. `json`, `xml`, etc you can try out [gomodifytags](https://github.com/fatih/gomodifytags).
+
+For example,
+
+```
+gomodifytags -file fileName -struct structName -add-tags json -w --skip-unexported
+```
+
+will add `json` tags for each struct generated.
+
```

Comment:
Good start, exactly the example I wanted 👍  But let me add this later the day 🙇

Reply:
Sure, please edit this as much as you want 😄

Reply:
Sry pretty busy here - can you remove that from the README again? I will add this later with CLI example 👍 Thanks!